scripts/gha/summarize_test_results.py

```diff
@@ -259,7 +259,7 @@ def main(argv):
         log_name[1] = "**%s" % log_name[1]
       with open(log_file, "r") as log_reader:
         log_reader_data = log_reader.read()
-        if "Android" in log_name or "iOS" in log_name:
+        if "Android" in log_name[1] or "iOS" in log_name[1]:
```

Comment:
We should do some sanitary checks on the list as we are assuming log_name[1] and log_name[2:] exists.

Reply:
Do you think we can add a quick check for this and exit with a useful error message if log_name doesn't have expected number of items?